rpcs3/rpcs3.vcxproj

```diff
@@ -166,7 +166,7 @@
     </ResourceCompile>
     <PostBuildEvent>
       <Command>
-        $(QTDIR)\bin\windeployqt --no-angle --no-opengl-sw --no-svg --no-translations --no-quick --plugindir $(TargetDir)\qt\plugins $(TargetPath)
```

Comment:
Does `$(TargetDir)` always contain a trailing backslash?

Reply:
It should: https://msdn.microsoft.com/en-us/library/c02as0cs.aspx

Reply:
I understand, thanks for the reference. The change seems correct.